### ISSUE TITLE
masterless: missing directories under /trinity 

### ISSUE THREAD

**Issue by themisa** [clustervision/trinity#379]
11ad056e 
masterless controller is missing the following directories:

/trinity: api+dash, clustervision, trinity_login

and version file

```
[root@controller() trinity]# ls
cinder  compute  container  login  nova  openstack  report
```

in other branches:
```
[root@controller(amaster) trinity]# ls
api+dash  clustervision  container  nova       report  trinity_login
cinder    compute        login      openstack  site    version
```

**Comment by abhimukh** [comment]
api+dash and trinity_login should not be under /trinity from r5 onwards (when we switched to the rootimg profiles).

/trinity/clustervision and /trinity/version get copied from the master. Since there is no master in masterless, there are not there!

**Comment by hansthen** [comment]
/trinity/version should still be created.

**Comment by Marc69** [comment]
Resolved in 1aee327; version file exists
